Makefile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env make run
+
+# Makefile information:
+#
+# Dependencies:
+#	Libraries: GL, X11, Xfixes, asound, m, pthread
+#	Executables: env, echo, rm, echo, sed, gcc, git
+#	Subprojects: git:github.com/w23/atto.git
+#
+# Produces:
+#	Executables: proto
+#	Pre-compiled objects: proto.c.o
+#	Subproject pre-compiled objects: atto/src/app_linux.c.o atto/src/app_x11.c.o
+
+# Makefile predefined variables:
+
+LDFLAGS :=-lGL -lX11 -lXfixes -lasound -pthread -lm
+INCLUDES:=-Iatto/include
+.SILENT : clean distclean atto/src/app_linux.c atto/src/app_x11.c
+.DEFAULT: run
+
+# run (or default) target calls proto target
+run: proto
+
+# this targets detects list of compilable files with includings and makes objects
+proto.c.o atto/src/app_linux.c.o atto/src/app_x11.c.o: proto.c atto/src/app_linux.c atto/src/app_x11.c
```

Comment:
Build artefacts should be in a separate directory. See makefiles from atto, OpenSource, stameska, etc..